vignettes/dm-class-and-basic-operations.Rmd

```diff
@@ -114,7 +136,8 @@ Some useful functions for managing primary key settings are:
 Currently `dm` objects only support one-column primary keys. If your tables have unique compound keys, adding a surrogate key column might be helpful. 
 If you created a `dm` object according to the examples in ["Examples of `dm` objects"](#ex_dm), your object does not yet have any primary keys set.
 So let's add one.
-By default, `cdm_add_pk()` checks if the column of the table given by the user is a unique key.
+
+`cdm_add_pk()` has an option to check if the column of the table given by the user is a unique key (for performance reasons check is turned off by default).
```

Comment:
for performance reasons, the check is turned off by default
OR
to speed up processing, the check will not be executed, unless requested